config/locales/two_factor_authentication/es.yml

```
@@ -1,6 +1,12 @@
 ---
 es:
   two_factor_authentication:
+    aal2_request:
+      phishing_resistant_html: '%{sp_name} requiere un método de autenticación de alta
+        seguridad, como el desbloqueo facial o táctil, una llave de seguridad o
+        una identificación de empleado público.'
+      piv_cac_only_html: '%{sp_name} requiere su identificación de empleado público,
```

do we want to add the `<strong>` tags like we have in English?
```suggestion
      phishing_resistant_html: '<strong>%{sp_name}</strong> requiere un método de autenticación de alta
        seguridad, como el desbloqueo facial o táctil, una llave de seguridad o
        una identificación de empleado público.'
      piv_cac_only_html: '<strong>%{sp_name}</strong> requiere su identificación de empleado público,
```

---

Yes we do! Good catch.

Also an interesting idea for a linter 🤔

---

no!!! too many lints

Unless.... make we make sure count of HTML tags matches in each locale? like # of `<strong>`, `<li>` etc etc

---

> Unless.... make we make sure count of HTML tags matches in each locale? like # of `<strong>`, `<li>` etc etc

Yes, that's what I had in mind. Similar to [what we check for with interpolation arguments](https://github.com/18F/identity-idp/blob/e17f5ba99fe52ceada90dfc882ae4a6e8e22b932/spec/i18n_spec.rb#L90-L108).

---

Fixed missing tags in f006460.

PR for linter: #8847